manager/machine.go

```diff
@@ -97,9 +103,9 @@ func getMachineInfo(sysFs sysfs.SysFs, fsInfo fs.FsInfo) (*info.MachineInfo, err
 		DiskMap:        diskMap,
 		NetworkDevices: netDevices,
 		Topology:       topology,
-		MachineID:      getInfoFromFiles(*machineIdFilePath),
+		MachineID:      getInfoFromFiles(path.Join(rootFs, *machineIdFilePath)),
 		SystemUUID:     systemUUID,
-		BootID:         getInfoFromFiles(*bootIdFilePath),
+		BootID:         getInfoFromFiles(path.Join(rootFs, *bootIdFilePath)),
```

Comment:
How about using `path/filepath` instead of `path`?  `path` is intended for forward slash-separated paths only (such as those used in URLs), while `path/filepath` manipulates paths across different operating systems, although the cadvisor only support linux system.